Scripts/Engine/Util.lua

```diff
@@ -6,77 +6,4 @@ Util.ShortenTENCalls = function()
 	print("Util.ShortenTENCalls is deprecated; its functionality is now performed automatically by TombEngine.")
 end
 
--- Check if the time format is correct.
```

Comment:
Why these three functions are removed from Util.lua? Where are the replacements?